docs/guides/layers.rst

```diff
@@ -481,11 +489,12 @@ pyramided. That is, we create a level-of-detail hierarchy that covers
 the same geographical extent, while each level of the pyramid uses one
 quarter as many pixels as the next level. This allows us to zoom in and
 out when the layer is being displayed without using extraneous detail.
-The ``pyramid`` method will produce an instance of ``Pyramid`` that will
-contain within it multiple ``TiledRasterLayer``\ s. Each layer
-corresponds to a zoom level, and the number of levels depends on the
-``zoom_level`` of the source layer. With the max zoom of the ``Pyramid``
-being the source layer's ``zoom_level``, and the lowest zoom being 0.
+The :meth:`~geopyspark.geotrellis.layer.TiledRasterLayer.pyramid` method
+will produce an instance of ``Pyramid`` that will contain within it multiple
```

Comment:
Should Pyramid and TiledRasterLayer be links here?  Not saying they certainly should.  Just a question, really.

Reply:
`TiledRasterLayer` was linked to earlier in the docs, so I didn't think it needed to be here. `Pyramid` should become a link.